### PR TITLE
MCP 2180: Documents unavailable for review

### PR DESCRIPTION
**Pull request by jbregstone**
## What was the problem?

MAS integration was not set up to accept information related to `documents unavailable for review`.

Associated tickets or Slack threads:

- [MCP-2180](https://amida.atlassian.net/browse/MCP-2180)

## How does this fix it?[^1]

Additional fields added to MAS integration for `documents unavailable for review`. 

## How to test this PR
- Step 1 
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: service-python/assessclaimdc7101/src/lib/queues.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def pdf_evidence_request_callback(channel, method, properties, body):
 
     binding_key = method.routing_key
     message = json.loads(body.decode("utf-8"))
+    logging.info(message)
```
Comment (dimitri-amida) [inline service-python/assessclaimdc7101/src/lib/queues.py:18]: Do we know this has no PII?

File: service-python/pdfgenerator/src/lib/queues.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def on_generate_callback(channel, method, properties, body):
         pdf_generator = PDFGenerator(pdf_options)
 
         message = json.loads(body)
-        # logging.info(f" [x] {binding_key}: Received message: {message}")
+        logging.info(f" Received message: {message}")
```
Comment (dimitri-amida) [inline service-python/pdfgenerator/src/lib/queues.py:23]: PII?

File: shared/domain/src/main/java/gov/va/vro/model/mas/MasAutomatedClaimPayload.java [hunk 3]
```diff
@@ -49,6 +49,14 @@ public class MasAutomatedClaimPayload implements Auditable {
 
   private List<String> veteranFlashIds;
 
+  @Schema(description = "Documents unavailable for automated review")
+  private List<String> documentsWithoutAnnotationsChecked;
+
+  /**
+   * Get diagnostic code.
```
Comment (dimitri-amida) [inline shared/domain/src/main/java/gov/va/vro/model/mas/MasAutomatedClaimPayload.java:56]: Not necessary for a method called getDiagnosticCode

**Review by dimitri-amida — COMMENTED** [review]
I have some comments about potential PII. This is still WIP and failing tests, so it will need another pass.

File: service-python/assessclaimdc7101/src/lib/queues.py (hunk 1 above)
Comment (jbregstone) [reply service-python/assessclaimdc7101/src/lib/queues.py:18]: I need to remove this

File: service-python/pdfgenerator/src/lib/queues.py (hunk 2 above)
Comment (jbregstone) [reply service-python/pdfgenerator/src/lib/queues.py:23]: Need to be deleted or commented

**Comment by jbregstone** [comment]
> No issues building/generating PDF. Did you have a ICN that has unavailable documents?

Unfortunately I do not know of a test veteran that we can use to test the end to end integration of this feature. We may have to earmark it for now. My understanding is that there are ongoing discussions on the test veterans we have access to.

**Comment by jbregstone** [comment]
#811

**Review by au2 — CHANGES_REQUESTED** [review]
This does not really work. I pushed an example in end to end test which uses mock collection id'ed 376.  Looking at the Java code.

**Review by rajesh-Krishnasamy — APPROVED** [review]

**Pull request merged**